warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/ProtobufEdgeDataTypeHandler.java

```diff
@@ -809,6 +830,9 @@ protected EdgeDataBundle createEdge(EdgeDefinition edgeDef, RawRecordContainer e
         }
         
         String typeName = event.getDataType().typeName();
+        String outputName = event.getDataType().outputName();
+        
+        typeName = edgeTypeLookup.containsKey(outputName) ? outputName : typeName;
```

Comment:
This sequence looks similar to the one above.  Perhaps refactor into a common method.

Reply:
Sure